db/migrate/20150923150410_add_times_purchased_to_book.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimesPurchasedToBook < ActiveRecord::Migration
+  def change
+    add_column :books, :times_purchased, :integer
```

Comment:
Bonus points if you use a `counter_cache`, but that's for later anyway after `Purchase` exists as a model.